Makefile

```diff
@@ -1,7 +1,6 @@
 # Define the target to run if make is called with no arguments.
 default: check
 
-export LOG_LEVEL?=fatal
```

Comment:
I recently added a change to set this to "9" in either another PR or commit... as this is useful IMO. maybe we should consider lowing the level and still make it overridable

Reply:
We don't need to set it in Makefile for it to be overridable, whoever runs make can set in their env. I suggest we use the default by default (i.e. don't set it), and if someone wants noisy tests they can `LOG_LEVEL=9 make check` or whatever.